setup.py

```diff
@@ -30,7 +30,7 @@
     "transformers",
     "scipy",
     "scikit-learn",
-    "deepspeed<0.13.0",
```

Comment:
My only fear is if deepspeed has fixed the issues that required us to pin it. @pacman100 do you know?

Reply:
Yes I think so, https://github.com/microsoft/DeepSpeed/issues/4984 is closed now.